boards/FSM/Src/App/states/App_AllStates.c

```diff
@@ -28,5 +31,25 @@ void App_AllStatesRunOnTick100Hz(struct StateMachine *const state_machine)
         App_CanRx_BMS_VITALS_SetSignal_HEARTBEAT(can_rx, false);
     }
 
+    // Check for torque plausibility
```

Comment:
I think this code should happen at 100Hz, or else will become outdated and false trip.

Reply:
It is I believe. It is running in `App_AllStatesRunOnTick100Hz`

Reply:
Indeed it is, my bad